api/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/cheqd/cheqd-node/api
```

Comment:
Maybe it's not required but should we call `lint` or `go build` for generated source code?